docs/routers/fritzbox.md

```diff
@@ -85,3 +84,90 @@ The following settings must be made:
     * **Local domain name (optional):** Fritz!Box uses **fritz.box**
 
 ![Screenshot der Conditional Forwarding Einstellungen](../images/conditional-forwarding.png)
+
+## Distribute Pi-hole as DNS server via IPv6
+
+Using this configuration, all clients will get Pi-hole's IPv6 offered as DNS server when connected to your Fritz!Box.
```

Comment:
Client behaviour cannot be guarantueed - I'd prefer a wording that just states what the FB would do, e.g.:

Using this configuration, your Fritz!Box will offer Pi-hole's IPv6 as local DNS server to its clients via DHCPv6 as well as Router Advertisements (RA, NDP/RDNSS).